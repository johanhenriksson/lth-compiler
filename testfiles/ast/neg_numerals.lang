int main() {
    a = -3;
    b = -3.5;
}
