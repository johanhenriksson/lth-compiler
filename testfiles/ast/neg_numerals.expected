Program
  List
    Function
      IdDecl
        TypeId
      List
      Block
        List
          AssignStmt
            IdUse
            IntLiteral
          AssignStmt
            IdUse
            FloatLiteral
